packages/typescript-resolver-files/src/utils/index.ts

```diff
@@ -5,10 +5,18 @@ export * from './isNativeNamedType';
 export * from './isRootObjectType';
 export * from './printImportLine';
 
+function slash(path:string) => string {
+  const isExtendedLengthPath: boolean = /^\\\\\?\\/.test(path);
+
+  if(isExtendedLengthPath) return;
+
+  return path.replace(/\\/g,'/');
+}
+
 // TODO: break the functions below this line into smaller files
 // -----------
 export const relativeModulePath = (from: string, to: string): string => {
-  const rawPath = path.relative(from, to);
+  const rawPath = slash(path.relative(from, to));
```

Comment:
I'm not too familiar with how this works on Windows. Can you please help me understand what the issue is here? :)

Reply:
Windows has problems with the \\ paths. Windows does better with the normal ones. So \ and /

Reply:
Yea to be more specific, windows uses a backslash as the path separator by default, so "\\" as  a string (escaped) or as a path "c:\Windows\...". The system-specific separator is returned by path.sep (Per docs: "\\" on Windows, "/" on Unix). Using path.relative uses / replaces all separators by system-specific separators, or resolves all paths as system specific paths, so \ on Windows. ESM-Imports are per spec only designed to work with unix paths, so those with forward slashes "/". The method (defined above) just replaces all occurrences of "\\" in windows paths as returned by path.relative with the forward slash used by unix paths. I hope that clears that up.

Reply:
> Windows has problems with the \\ paths. Windows does better with the normal ones. So \ and /

That is not quite true. Windows (sometimes) supports using unix paths as well, but the native format is still the backslash-separated one - Javascript instead does not support windows paths as import paths. Some packages (like glob i believe) have issues with using windows paths even on windows systems so they default to unix paths as well - I know we had to implement a fix for that in one of your projects (@devpanda0) but that is a separate issue from the one here.

Reply:
Thanks for the explanation 🙌  @JustusFluegel 

I'm curious if this path separator issue applies to users who use WSL?

Reply:
> I have tested it in WSL and it works fine

Thanks @devpanda0 ! If it's working in WSL, please continue using it for now. 

I'm having issues trying to set up Windows test in CI in this [PR](https://github.com/eddeee888/graphql-code-generator-plugins/pull/67). I'll come back to this later.